node/core/av-store/src/lib.rs

```diff
@@ -56,22 +56,36 @@ mod columns {
 
 #[derive(Debug, Error)]
 enum Error {
-	#[error(transparent)]
-	RuntimeAPI(#[from] RuntimeApiError),
 	#[error(transparent)]
 	ChainAPI(#[from] ChainApiError),
 	#[error(transparent)]
 	Erasure(#[from] erasure::Error),
 	#[error(transparent)]
 	Io(#[from] io::Error),
 	#[error(transparent)]
-	Oneshot(#[from] oneshot::Canceled),
+	ChainAPIChannelIsClosed(#[from] oneshot::Canceled),
 	#[error(transparent)]
 	Subsystem(#[from] SubsystemError),
 	#[error(transparent)]
 	Time(#[from] SystemTimeError),
 }
 
+/// Class of errors which we should handle more gracefully.
```

Comment:
If I had to handle this error I'd be curious what I can or cannot do with it. Can we mention that these must not lead to bringing down the whole subsystem?